### ISSUE TITLE
Stuns should not completely disable gameplay

### ISSUE THREAD

**Issue by greatguys1** [MSRevive/MSCScripts#52]
Currently, most stuns vastly reduce movement speed and stop the player from being able to attack. This is not fun to play against and is very frustrating, especially with enemies that like to stun constantly. I would like to incorporate stuns in a way that doesn't feel unfair against the player. I'm thinking either movement should be slowed, OR being unable to attack, but never both at the same time. Personally I wouldn't like to limit attacking if there is another idea.

**Comment by SaintWish** [comment]
Yeah maybe something like just slowing the character down by either 50% or 80% should be good like you said.

**Comment by greatguys1** [comment]
Yes, any effect that stops your ability to play is not fun. I will change this issue's name to be inclusive of things like freezing, and magic barriers.
Attack telegraphing is definitely possible, as you pointed out, some enemies already do this. That will be a lot more involved though, unlike changing the stun effect. We would probably have to see an prioritize more important encounters first.
